deployment-config/mydeploy.json

```diff
@@ -0,0 +1,61 @@
+{
+    "protocolAdmin": "0xC2d99d76bb9D46BF8Ec9449E4DfAE48C30CF0839",
+
+    "boringVault":{
+        "boringVaultSalt": "0x1ddd634c506ad203da17ff00000000000000000000000000000000000000000a",
+        "boringVaultName": "Ion Vault", 
+        "boringVaultSymbol": "IVT",
+    
+        "address": "0x0000000000E7Ab44153eEBEF2343ba5289F65dAC"    
+    },
+
+    "manager":{
+        "managerSalt": "0x30432d4b4ec00003b4a25000000000000000000000000000000000000000000a",
+
+        "address": "0x0000000000fAd6Db23abdC1a85621B97bd1Dc82f"
+    },
+    
+    "accountant":{
+        "accountantSalt": "0x6a184dbea6f3cc0318679f00000000000000000000000000000000000000000a",
+        "payoutAddress": "0x0000000000417626Ef34D62C4DC189b021603f2F",
+        "base": "0x5f207d42F869fd1c71d7f0f81a2A67Fc20FF7323",
+        "allowedExchangeRateChangeUpper": "10003",
+        "allowedExchangeRateChangeLower": "9998",
+        "minimumUpdateDelayInSeconds": "3600",
+        "managementFee": "2000",
+
+        "address": "0x00000000004F96C07B83e86600D86F9479bB43fa"
+    },
+
+    "teller": {
+        "tellerSalt": "0x51f8968749a56d01202c9100000000000000000000000000000000000000000a",
+        "maxGasForPeer": 100000,
+        "minGasForPeer": 0,
+        "peerEid": 0,
+        "tellerContractName": "MultiChainLayerZeroTellerWithMultiAssetSupport",
+        "opMessenger": "0x25ace71c97B33Cc4729CF772ae268934F7ab5fA1",
+        "lzEndpoint": "0x1a44076050125825900e736c501f859c50fE728c",
+
+        "address": "0x00000000004F96C07B83e86600D86F0000000000"
+    },
```

Comment:
Can we make it such that we can differentiate which configuration parameters apply to which type of tellers? Right now, it takes brainpower to tell which configuration should be populated when deploying a certain type of teller. And this would get more confusing as we add more teller types.